site/zenodo_rdm/migrator/scripts/dump_awards_db.py

```diff
@@ -0,0 +1,131 @@
+"""Parse OpenAIRE awards dumps into CSV format, importable via COPY.
+
+A modified version of https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/contrib/awards/datastreams.py
+for the purpose of producing an easy to load CSV dump of the awards into an InvenioRDM instance.
+
+To use call ``load_files(DATA_PATHS, "awards.csv")``.
+"""
+import csv
+import datetime as dt
+import gzip
+import json
+import uuid
+
+DATA_PATHS = [
+    "awards-apr-2023.jsonl.gz",  # https://zenodo.org/record/7870151
+    "awards-mar-2023.jsonl.gz",  # https://zenodo.org/record/7803150
+    "awards-feb-2023.jsonl.gz",  # https://zenodo.org/record/7683844
+    "awards-jan-2023.jsonl.gz",  # https://zenodo.org/record/7561801
+    "awards-dec-2022.jsonl.gz",  # https://zenodo.org/record/7745773
+]
+
+VOCABULARIES_AWARDS_OPENAIRE_FUNDERS = {
+    "aka_________": "05k73zm37",
+    "anr_________": "00rbzpz17",
+    "arc_________": "05mmh0f86",
+    "asap________": "03zj4c476",
+    "cihr________": "01gavpb45",
+    "corda_______": "00k4n6c32",
+    "corda_____he": "00k4n6c32",
+    "corda__h2020": "00k4n6c32",
+    "euenvagency_": "02k4b9v70",
+    "fct_________": "00snfqn58",
+    "fwf_________": "013tf3c58",
+    "inca________": "03m8vkq32",
+    "irb_hr______": "03n51vw80",
+    "mestd_______": "01znas443",
+    "nhmrc_______": "011kf5r70",
+    "nih_________": "01cwqze88",
+    "nserc_______": "01h531d29",
+    "nsf_________": "021nxhr62",
+    "nwo_________": "04jsz6e67",
+    "rcuk________": "00dq2kk65",
+    "sfi_________": "0271asj38",
+    "snsf________": "00yjd3n13",
+    "sshrc_______": "006cvnv84",
+    "tubitakf____": "04w9kkr77",
+    "ukri________": "001aqnf71",
+    "wt__________": "029chgv08",
+}
+VOCABULARIES_AWARDS_EC_ROR_ID = "00k4n6c32"
+
+
+def _ts():
```

Comment:
this is available in invenio-rdm-migrator (can be imported) https://github.com/inveniosoftware/invenio-rdm-migrator/blob/master/invenio_rdm_migrator/utils.py